pages/contact.tsx

```diff
@@ -0,0 +1,54 @@
+        </FAQ>
+
+        <FAQ anchorId="customise" question="Can I customise my order?">
+          <p>All our products state whether they have customisable elements. Unfortunately it is not possible for us to make custom changes on request, but feel free to suggest ideas and we may add these to our products in the future!</p>
+        </FAQ>
+
+        <FAQ anchorId="order-not-arrived" question="Why has my order not arrived yet?">
+          <p>Unfortunately Royal Mail is experiencing delays due to the coronavirus pandemic. If your order has not arrived in 10 working days, please contact us at <a href="mailto:support@roseatecards.co.uk">support@roseatecards.co.uk</a> or use our <a href="http://m.me/roseatecards">live chat on Facebook Messenger</a></p>
+        </FAQ>
+
+        <FAQ anchorId="wrong-order" question="I have received an incorrect or faulty item">
+          <p>We’re really sorry if you’ve experienced any problems with your order! Unfortunately we can’t accept returns, but if your item is incorrect, faulty or damaged then we can offer a refund. Please contact us at <a href="mailto:support@roseatecards.co.uk">support@roseatecards.co.uk</a> or use our <a href="http://m.me/roseatecards">live chat on Facebook Messenger</a></p>
+        </FAQ>
+
+        <FAQ anchorId="how-long" question="How long will my order take to print and ship?">
+          <p>Orders take approximately 1-2 days to print and 3-4 days for 2nd class delivery in the UK (please see the latest <a href="#covid19">COVID-19 update</a> for information on shipping delays.</p>
+        </FAQ>
+
+        <FAQ anchorId="tech" question="Trouble with the website itself">
+          <p>Oops, did we break something? Let us know by emailing <a href="mailto:tech@roseatecards.co.uk">tech@roseatecards.co.uk</a>, providing as much detail as possible about what isn’t working</p>
+        </FAQ>
+      </div>
+
+      <div className="contact__methods">
+        <h1 className="page-title">Contact Us</h1>
+
+        <a className="contact__methods__link" href="mailto:hello@roseatecards.co.uk">
+          <svg xmlns="http://www.w3.org/2000/svg" width="70px" height="55px" viewBox="0 0 122.88 71.9"><path d="M29.56 0h88.02c1.57 0 2.95.64 3.9 1.67.96 1.03 1.5 2.46 1.38 4.03l-4.42 60.49a6.107 6.107 0 01-1.22 3.2c-.14.3-.33.57-.59.8-.17.15-.36.28-.56.37-1.05.83-2.36 1.33-3.74 1.33H24.31c-1.57 0-2.95-.64-3.91-1.67-.96-1.03-1.49-2.46-1.38-4.03l1.25-10.18h4.37l-1.08 7.78 33.11-30.93-29-25.14-.29 3.93H23l.43-5.95c.11-1.57.86-3 1.97-4.03C26.51.64 27.99 0 29.56 0zM2.19 49.78a2.19 2.19 0 110-4.38h28.89a2.19 2.19 0 110 4.38H2.19zm10.83-13.56a2.19 2.19 0 110-4.38h25.06a2.19 2.19 0 110 4.38H13.02zM2.19 22.66a2.19 2.19 0 110-4.38h28.89a2.19 2.19 0 110 4.38H2.19zm57.68 12.98L25.74 67.52h85.53L83.2 35.64l-10.64 8.54c-.83.67-2.01.71-2.8.03l-9.89-8.57zm26.87-2.83l27.5 31.23 4.13-56.6-31.63 25.37zM30.71 4.37l40.73 35.31 44.03-35.31H30.71z"/></svg>
```

Comment:
This makes the code a little hard to read.
I would suggest doing one of the following:

- saving these as `.svg` files in `/public` and then using an `<img>`
- saving these as `.svg` files in `/components/icons/` and adding support to `import` `.svg` files as React components